doc/source/train/user-guides/data-loading-preprocessing.rst

```diff
@@ -325,6 +325,14 @@ For more details, see the following sections for each framework:
     Instatiating the Dataset outside of the ``train_loop_per_worker`` and passing it in via global scope
     can cause errors due to the Dataset not being serializable.
 
+.. note::
+
+    When using PyTorch DataLoader with more than 1 worker, you should set the
+    process start method to be `forkserver` or `spawn` i.e.
+    ``data_loader = DataLoader(dataset, multiprocessing_context=multiprocessing.get_context("forkserver"), num_workers=2...)``.
```

Comment:
nits: Would recommend formatting this as multi-line code to make it easier to read, and also put num_workers first since the multiprocessing-context is actually the additional part.

Something like:


```
.. code-block:: python

data_loader = DataLoader(
    dataset,
    num_workers=2, 
    multiprocessing_context=multiprocessing.get_context("forkserver"), 
    ...
)

Reply:
done:
![Screenshot 2025-05-14 at 10 27 15 AM](https://github.com/user-attachments/assets/86e0ce06-1d10-4d18-9c5c-e23c0ecb984c)